packages/nuxt/src/core/nuxt.ts

```diff
@@ -621,4 +628,15 @@ export async function loadNuxt (opts: LoadNuxtOptions): Promise<Nuxt> {
   return nuxt
 }
 
+async function checkDependencyVersion (name: string, nuxtVersion: string): Promise<void> {
+  const path = await resolvePath(name).catch(() => null)
+
+  if (!path) { return }
+  const { version } = await readPackageJSON(path)
+
+  if (version !== nuxtVersion) {
```

Comment:
shouldn't we accept _newer_ versions of kit + schema? We could use semver `gte` to check...